scripts/actions/mobskills/lamentation.lua

```diff
@@ -13,13 +13,17 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    local damage = mob:getWeaponDmg() * 3
-
-    damage = xi.mobskills.mobMagicalMove(mob, target, skill, damage, xi.element.LIGHT, 1, xi.mobskills.magicalTpBonus.NO_EFFECT)
+    local damage = xi.mobskills.mobMagicalMove(mob, target, skill, mob:getMainLvl() + 2, xi.element.LIGHT, 1, xi.mobskills.magicalTpBonus.NO_EFFECT)
     damage = xi.mobskills.mobFinalAdjustments(damage, mob, skill, target, xi.attackType.MAGICAL, xi.damageType.LIGHT, xi.mobskills.shadowBehavior.WIPE_SHADOWS)
 
     target:takeDamage(damage, mob, xi.attackType.MAGICAL, xi.damageType.LIGHT)
-    xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.DIA, 3, 3, 60)
+
+    xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.DIA, 8, 3, 30)
```

Comment:
Jimmy's sheet says this is about a 20.31% def reduction, so power should be 20 (dia effect uses DEFP which is integer only)

```suggestion
    xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.DIA, 8, 3, 20)
```

Reply:
30 is the duration here. IIRC for Bio/Dia, the secondary effect is stored in subPower.

So it would be:
`xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.DIA, 8, 3, 30, subType, subPower)`

Reply:
Adjusted subpower to pass the defense down percent.